presto-main/src/main/java/com/facebook/presto/sql/planner/RelationPlan.java

```diff
@@ -46,13 +43,6 @@ public RelationPlan(PlanNode root, Scope scope, List<Symbol> fieldMappings)
         this.fieldMappings = ImmutableList.copyOf(fieldMappings);
     }
 
-    public Optional<Symbol> getSymbol(Expression expression)
```

Comment:
maybe you can copy-paste part of your commit message as javadoc for this class to prevent adding such methods in future